crates/wasmi/src/store.rs

```diff
@@ -26,6 +28,8 @@ use crate::{
     TableEntity,
     TableIdx,
 };
+#[cfg(not(feature = "std"))]
```

Comment:
Due to https://github.com/paritytech/wasmi/blob/master/crates/wasmi/src/lib.rs#L85 we do not even need the `#[cfg(not(feature = "std"))]`. :) I know it isn't standard practice but it helps to get rid of tons of those `#[cfg]` attributes which are just noise at some point.

Reply:
Ok, ok. Removing.